### ISSUE TITLE
error occurs during execution

### ISSUE THREAD

**Issue by MaineK00n** [ehimetakahashilab/Logic-BIST-Sim#8]
After this fix(https://github.com/ehimetakahashilab/Logic-BIST-Sim/commit/3fec558c72363c5330da444fa5a9d32bd5e16b5a), I get segmentation fault. Are you sure it works?

```console
$ ./multi-CPI.sh
...
==simulation for Logic CPI ===
===s9234: 1 1===========
External Parameter Setting
TOOL_MODE=4
Tool: Logic CP Insertion MODE 4
TPG: 16 bit LFSR
./multi-CPI.sh: line 83:  5620 Segmentation fault      ./lbistsim $CIRCUIT $TOOLMODE $TPG $CP_CTRL $LCP_rate $CAPTURE $INTERVAL_CYCLE $SKIP_CAP $cnt1 $OBRATE $CP_GROUP $TEST_VEC
...
```

**Comment by MaineK00n** [comment]
Execute `Instance_Get` in the main function and access `flt_det_num` inside it, if `TOOL_MODE=4`, but the array is not allocated at that time.

**Comment by MaineK00n** [comment]
fix in 747ce6033a169f3f03f8352daf633915f857dd05
